modules/http/pom.xml

```diff
@@ -148,6 +148,12 @@
             <version>4.3.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpasyncclient</artifactId>
+            <version>4.0.2</version>
```

Comment:
Do we not use this anywhere else, if we do then ideally version should be in base pom.xml